test/parallel_testsuite.py

```diff
@@ -115,6 +115,13 @@ def printResult(res):
   # working directory is not within it.
   os.chdir(olddir)
   common.force_delete_dir(temp_dir)
+
+  # Since we are returning this result to the main thread we need to make sure
+  # that it is serializable/picklable. To do this, we delete any non-pickalble
```

Comment:
```suggestion
  # that it is serializable/picklable. To do this, we delete any non-picklable
```
(not that this is a real word, no idea how to spell it :smile: but this makes us more consistent internally)